src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -102,6 +103,7 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
     resolveAndCleanup: useRef<() => void>(),
   };
   const currentInteractiveState = useRef<any>(initialInteractiveState);
+  const hasHeaderClass = hasHeader ? "iframe-has-header" : "iframe";
```

Comment:
1. hasHeader is a number, but clearly used as a boolean. Note that if `quesitonNumber` is 0, it might not work as expected (it might never not equal to 0, but who knows where do we increase index to start from 1).
2. "iframe" class doesn't seem to be used anywhere.
3. Note that we have a `classname` lib in AP. You could use it like `classnames({ "has-header": hasHeader });`.

Reply:
Fix in follow on PRs.